client/packages/system/src/Encounter/DetailView/Footer.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { FC } from 'react';
+import {
+  Box,
+  useTranslation,
+  AppFooterPortal,
+  ButtonWithIcon,
+  CheckIcon,
+  XCircleIcon,
+  useNavigate,
+  Typography,
+  DetailContainer,
+  DialogButton,
+  ClockIcon,
+  useDialog,
+  StatusCrumbs,
+  EncounterNodeStatus,
+} from '@openmsupply-client/common';
+import {
+  DocumentHistory,
+  EncounterFragment,
+} from '@openmsupply-client/programs';
+import { encounterStatusTranslation } from '../utils';
+
+interface FooterProps {
+  documentName?: string;
+  isDisabled: boolean;
+  onCancel: () => void;
+  onSave: () => void;
+  encounter?: EncounterFragment;
+}
+
+const EncounterStatusCrumbs: FC<{ encounter: EncounterFragment }> = ({
+  encounter,
+}) => {
+  const t = useTranslation('common');
+  // TODO StatusCrumbs shows "Order History" in the logs pop up -> make this configurable
```

Comment:
🤔

Reply:
The way StatusCrumbs is used this doesn't show up here. The StatusCrumbs has a `<PaperPopoverSection label={t('label.order-history')}>` component, this can be lifted out of the StatusCrumbs...